### ISSUE TITLE
about model tool

### ISSUE THREAD

**Issue by bikemonx** [SydMontague/DSCSTools#2]
can you make tools to extract models and effect files?

**Comment by SydMontague** [comment]
I haven't done any work or research regarding that topic yet and doubt I'll find the time to do so in the near future.

Regarding tools for models you may want to look into this project, though:
https://github.com/xdanieldzd/CSGeom

It was for the PS4/PSVita releases of the game but I don't think the relevant files changed their format in the port.

**Comment by bikemonx** [comment]
Ok，thanks. 

How did you know the file format? By analyzing binary files or using tools like ida?

**Comment by SydMontague** [comment]
People were able to extract the archive format for the PS4/PSVita versions before (and more importantly, for other games of the same company) so I could build on their work without having to figure out the entire format myself.

(-> https://forum.xentax.com/viewtopic.php?f=10&t=12666)

But yeah, the stuff that was missing (most importantly everything needed to re-build them) I had to figure out by looking at the hex dump. Not sure if I used a debugger at some point to learn some more, though.
